packages/did-document/package.json

```diff
@@ -36,7 +36,7 @@
     "ethers": "^5.7.2"
   },
   "devDependencies": {
-    "eslint": "^7.1.0",
+    "eslint": "^8.6.0",
```

Comment:
if `did-document` uses `eslint` different from hoisted, it results in false error